apps/api/src/modules/auth/auth.service.ts

```diff
@@ -68,12 +68,12 @@ export class AuthService {
     return this.createJwtToken(user, res)
   }
 
-  async createJwtToken(user: User, res: Response) {
+  createJwtToken(user: User, res: Response) {
     const { userId, type } = user
     const token: string = this.jwtService.sign({ userId, type })
     res.cookie('token', token, {
       httpOnly: true,
-      secure: this.configService.get<boolean>('cookie.secure'),
+      secure: false,
```

Comment:
Why we change it back to `false` ?

Reply:
A...
I forgot to change back, because I have a problem while testing using this service